packages/rum/test/benchmarks/profiler.js

```diff
@@ -23,43 +23,51 @@
  *
  */
 
-const puppeteer = require('puppeteer')
+const playwright = require('playwright')
 const { chrome } = require('./config')
 const {
   filterCpuMetrics,
   capturePayloadInfo,
   getMemoryAllocationPerFunction
 } = require('./analyzer')
 
-async function launchBrowser() {
-  return await puppeteer.launch(chrome.launchOptions)
+async function launchBrowser(type) {
+  return await playwright[type].launch(chrome.launchOptions)
 }
 
 function gatherRawMetrics(browser, url) {
   return new Promise(async resolve => {
     /**
      * Create a separate browsing context for each run
      */
-    const context = await browser.createIncognitoBrowserContext()
+    const context = await browser.newContext()
     const page = await context.newPage()
-    const client = await page.target().createCDPSession()
+
+    let client = null
+    try {
+      client = await browser.pageTarget(page).createCDPSession()
```

Comment:
Why this might not be available? probably a comment would be useful

Reply:
Sure, will do.